cypress/integration/requests/patron-initiated/request-item-recall.spec.js

```diff
@@ -0,0 +1,162 @@
+import DevTeams from '../../../support/dictionary/devTeams';
+import TestTypes from '../../../support/dictionary/testTypes';
+import TopMenu from '../../../support/fragments/topMenu';
+import RequestPolicy, { defaultRequestPolicy } from '../../../support/fragments/circulation/request-policy';
+import PatronGroups from '../../../support/fragments/settings/users/patronGroups';
+import Users from '../../../support/fragments/users/users';
+import DefaultUser from '../../../support/fragments/users/userDefaultObjects/defaultUser';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../../support/fragments/users/userEdit';
+import InventoryHoldings from '../../../support/fragments/inventory/holdings/inventoryHoldings';
+import CirculationRules from '../../../support/fragments/circulation/circulation-rules';
+import Checkout from '../../../support/fragments/checkout/checkout';
+import CheckInActions from '../../../support/fragments/check-in-actions/checkInActions';
+import Requests from '../../../support/fragments/requests/requests';
+import NewRequest from '../../../support/fragments/requests/newRequest';
+
+describe('ui-requests: Request: Edit requests. Make sure that edits are being saved.', () => {
+  // TODO: A client configured to use edge-patron API
+  const requestPolicyWithRecall = defaultRequestPolicy;
+  const testData = {
+    instanceTitle: `For_request_${Number(new Date())}`,
+    pickupServicePoint: 'Circ Desk 1',
+  };
+  const patronGroup = {};
+  const userData0 = { ...DefaultUser.defaultApiPatron };
+  const userData1 = { ...DefaultUser.defaultApiPatron };
+  const ITEM_BARCODE = generateItemBarcode();
+  userData1.username = `autotest_username_${getRandomPostfix()}`;
+  userData1.barcode = `1234456_${getRandomPostfix()}`;
+  requestPolicyWithRecall.requestTypes.push('Recall', 'Page');
+
+  console.log(requestPolicyWithRecall);
+
+  before('Creating circ rule with request policy, users and item with rolling loan period', () => {
+    cy.getAdminToken();
+
+    // Create request policy that allows recalls and pages
+    RequestPolicy.createApi(requestPolicyWithRecall).then((body) => {
+      testData.requestPolicyId = body.id;
+    });
+
+    // Create 2 users in the same group
+    PatronGroups.createViaApi()
+      .then(res => {
+        patronGroup.id = res;
+        Users.createViaApi({
+          patronGroup: res,
+          ...userData0
+        }).then((createdUser) => {
+          userData0.id = createdUser.id;
+        });
+
+        Users.createViaApi({
+          patronGroup: res,
+          ...userData1
+        }).then((createdUser) => {
+          userData1.id = createdUser.id;
+        });
+      });
+
+    // Create item with remembered instance type
+    ServicePoints.getViaApi({ limit: 1, query: 'pickupLocation=="true"' })
```

Comment:
try to use InventoryInstances.createInstanceViaApi
Or you can create your own creating function

Reply:
done)